R/getTemplate.R

```diff
@@ -1,11 +1,21 @@
-#' Retrieves latest template for a given project or mapping file
+#' getTemplate
+#'
+#' Retrieves latest template for a given project.
+#' Templates must contain the columns "Variable", "Unit", "piam_variable",
```

Comment:
see above: shouldn't we use lowercase everywhere?

Reply:
done